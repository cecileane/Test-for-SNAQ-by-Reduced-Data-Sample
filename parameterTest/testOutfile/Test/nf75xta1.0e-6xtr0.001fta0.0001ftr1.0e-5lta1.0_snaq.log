optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.0001,
                       xtolAbs=1.0e-6, xtolRel=0.001.
 max number of failed proposals = 75, liktolAbs = 1.0.
rootname for files: nf75xta1.0e-6xtr0.001fta0.0001ftr1.0e-5lta1.0_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 94193
seed: 94193 for run 1
Sat Oct  1 17:01:23 2016

 BEGIN SNaQ for run 1, seed 94193 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 94 (failures: 75) with -loglik=128.22643 and ht_min=[0.30771,0.66107,0.4599,1.7391,8.87903]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 5		 5		 1	 |	 NA 		 NA 		 NA 
mvorigin	 25		 25		 0	 |	 0		 0		 --
mvtarget	 25		 25		 0	 |	 0		 0		 --
chdir		 4		 4		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 35		 29		 1	 |	 6		 6		 --
Total		 94		 88		 2	 |	 6		 6		 6
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 1, -loglik of best 128.22642794678467
(((1,#H7:::0.307713191038884):8.879027973078074,2):0.6610738299275306,4,((5,(6)#H7:::0.692286808961116):1.7390959980604472,3):0.45989549278037944);
---------------------
seed: 63503 for run 2
Sat Oct  1 17:01:31 2016

 BEGIN SNaQ for run 2, seed 63503 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 113 (failures: 75) with -loglik=155.49626 and ht_min=[0.33535,0.12573,0.55437,0.32983]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 33		 33		 2	 |	 0		 0		 --
mvtarget	 35		 35		 1	 |	 0		 0		 --
chdir		 9		 9		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 35		 29		 3	 |	 1		 1		 --
Total		 113		 107		 8	 |	 1		 1		 1
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 2, -loglik of best 155.49625734557495
(5,6,(((4,#H7:::0.37302464344577446):2.155860915831032,(1,(2)#H7:::0.6269753565542255):2.155860915831032):0.12573267863275725,3):0.33534740761821785);
---------------------

Sat Oct  1 17:01:36 2016
MaxNet is (((1,#H7:::0.307713191038884):8.879027973078074,2):0.6610738299275306,4,((5,(6)#H7:::0.692286808961116):1.7390959980604472,3):0.45989549278037944); 
with -loglik 128.22642794678467
