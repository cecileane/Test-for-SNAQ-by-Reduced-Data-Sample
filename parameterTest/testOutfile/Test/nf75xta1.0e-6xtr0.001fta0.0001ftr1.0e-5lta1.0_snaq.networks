(((1,#H7:::0.307713191038884):8.879027973078074,2):0.6610738299275306,4,((5,(6)#H7:::0.692286808961116):1.7390959980604472,3):0.45989549278037944);, with -loglik 128.22642794678467 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(((1)#H7:::0.6363174354788832,2):3.3939256170820955,4,((5,(6,#H7:::0.36368256452111686):1.1664446947053904):0.4361682583001232,3):0.5408390462759429);, with -loglik 130.3438072153437
((1,(6,(((4)#H7:::0.623597311554116,3):0.024837503977067663,5):0.211393734906151):1.1755335894507082):9.186362999292735,#H7:::0.376402688445884,2);, with -loglik 171.9511005887953
(#H7:::0.41537762949796186,4,((5,(6,(1,(2)#H7:::0.5846223705020381):3.4519653515219413):0.16887042397008423):0.03713569049835426,3):1.938603737895688);, with -loglik 192.56105742928918
(((1,(6,(#H7:::0.46815600681310743,5):1.207449999778451):0.9946480852672096):0.1122508914442501,2):0.1303129529897808,4,(3)#H7:::0.5318439931868926);, with -loglik 215.1560827456505
(((1,(6,#H7:::0.49587555381114534):3.9496123392604066):0.0465531298976689,2):0.20452983049648085,4,((5)#H7:::0.5041244461888547,3):1.175602415290075);, with -loglik 231.82835581136086
