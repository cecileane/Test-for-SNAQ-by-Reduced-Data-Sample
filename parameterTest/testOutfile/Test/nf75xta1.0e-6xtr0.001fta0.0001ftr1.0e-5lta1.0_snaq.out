(((1,#H7:::0.307713191038884):8.879027973078074,2):0.6610738299275306,4,((5,(6)#H7:::0.692286808961116):1.7390959980604472,3):0.45989549278037944); -Ploglik = 128.22642794678467
 Dendroscope: (((1,#H7):8.879027973078074,2):0.6610738299275306,4,((5,(6)#H7):1.7390959980604472,3):0.45989549278037944);
 Elapsed time: 12.872035245 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (((1,#H7:::0.307713191038884):8.879027973078074,2):0.6610738299275306,4,((5,(6)#H7:::0.692286808961116):1.7390959980604472,3):0.45989549278037944);, with -loglik 128.22642794678467
 (5,6,(((4,#H7:::0.37302464344577446):2.155860915831032,(1,(2)#H7:::0.6269753565542255):2.155860915831032):0.12573267863275725,3):0.33534740761821785);, with -loglik 155.49625734557495
-------
